docs/edge.rst

```diff
@@ -289,7 +306,7 @@ deletion of a custom region disables access to CrateDB Cloud for that region.
 To confirm deletion of the custom region, enter the name of your region into
 the form.
 
-.. image:: _assets/img/cloud-edge-deletion.png
```

Comment:
Would you like to remove the old image altogether?

Reply:
Yeah I clean them up usually. In this case I'm keeping it briefly, just in case my attempted resizing does not work. Saves a little bit of uploading tedium